### DISCUSSION TITLE
Keep stopped containers that were made with quadlets listed in the Podman GUI

### DISCUSSION THREAD

**Discussion by Vandaahl** [cockpit-project/cockpit#21222] [Q&A] [answered]
I have started using Podman quadlets to create my Podman containers instead of Podman Compose. It works well, but I when I use the Podman Container GUI of Cockpit to stop them, they are destroyed and removed from the list. Is there a way to prevent this from happening? One way I know of is setting [Service] Restart=always in the quadlet container file. But I don't like that behaviour because I want a stopped container to be stopped, not immediately recreated and started.

**Comment by tomasmatus** [comment, marked as answer]
If I recall correctly this is the way quadlets behave. So when you `podman stop ...` a quadlet container it is completely brought down and destroyed. You can check if podman can still see the container with `podman ps -a` after you stop it and if it can't then there's not much we can do about it in cockpit.

**Reply by Vandaahl** [reply]
Yeah I was afraid that would be the case. Indeed the container is gone when you stop it.

**Reply by almereyda** [reply]
This does not necessarily need to be a bad thing and can also be read as an incentive to properly separate immutable container image state from mutable application state.

**Reply by jelly** [reply]
A newer version of cockpit-podman lists inactive quadlets which we discover by scanning /run/systemd/

@almereyda please don't necro bump this thread.
